backend/src/platform/testManager/core.py

```
@@ -163,13 +162,13 @@ def get_test_suite_for_test(
             return None
 
         if suite.visibility == "private":
-            require_resource_access(principal, suite.owner)
+            require_resource_access(principal_id, suite.owner)
         return suite
 
-    def get_test(self, session: Session, principal: Principal, test_id: str) -> Test:
+    def get_test(self, session: Session, principal_id: str, test_id: str) -> Test:
         test = session.query(Test).filter(Test.id == test_id).one_or_none()
         if test is None:
             raise ValueError("test not found")
```

Ruff (0.14.2)

Avoid specifying long messages outside the exception class

(TRY003)